android/app/src/main/java/com/bounswe2020group3/paperlayer/util/Constants.kt

```diff
@@ -1,5 +1,5 @@
 package com.bounswe2020group3.paperlayer.util
 
-//const val BASE_URL = "https://paperlayer.herokuapp.com"
-const val BASE_URL = "https://paperlayer-backend.azurewebsites.net"
-const val TOKEN = "Token 94636236e607c560cb13333e4b60b9faa5bc125d"
\ No newline at end of file
+const val BASE_URL = "https://paperlayer.herokuapp.com"
+//const val BASE_URL = "https://paperlayer-backend.azurewebsites.net"
```

Comment:
Just like I commented on @mhrfky 's pull requests, I think we should keep the azure in the master